client/components/editor/structure/ContentContainers/Container.vue

```diff
@@ -19,27 +19,41 @@
       Click the button below to create content.
     </v-alert>
     <element-list
-      @add="addElement"
-      @insert="insert"
       @update="reorder"
-      :list="contentElements"
+      :elements="contentElements"
       :activity="container"
-      :types="types"
-      :layout="layout">
-      <template v-slot:list-item="{ item, dragged, setWidth }">
+      :supported-types="types"
+      :layout="layout"
+      class="element-list">
+      <template v-slot:list-item="{ element, isDragged, position }">
+        <inline-activator @click.native="showElementDrawer(position - 1)" />
         <content-element
```

Comment:
```suggestion
	<content-element v-bind="{ isDragged, element, setWidth: false }" />
```
It's a preferable thing 🙃